crates/e2e/src/setup/deploy.rs

```diff
@@ -40,6 +42,11 @@ impl Contracts {
         tracing::info!("connected to forked test network {}", network_id);
 
         let gp_settlement = GPv2Settlement::deployed(web3).await.unwrap();
+        let cow_amm_helper = match contracts::CowAmmLegacyHelper::deployed(web3).await {
+            Err(DeployError::NotFound(_)) => None,
+            Err(err) => panic!("failed to find deployed contract: {:?}", err),
+            Ok(contract) => Some(contract),
```

Comment:
nit: this is duplicated across the crates. How about defining an "unsafe" `deploy` function?